### ISSUE TITLE
CR: Adding a way to contact the municipality

### ISSUE THREAD

**Issue by SjoerdHilhorst** [teamforus/general#498]
Learn more about change requests here: https://bit.ly/39CWeEE

### Requested by:
- Paul en Leonie  
Note: This change was requested after acceptance testing with stakeholders of project 100

### Change description
- As a user i would like to contact forus or the municipality via a contact form or chat box
![image](https://user-images.githubusercontent.com/48722300/95092741-499c5300-0728-11eb-9e8c-e0356f0380fa.png)
Currently the ways to contact are very minimal

**Comment by maxvisser** [comment]
@GerbenBosschieter Could you enhance this feature request further? Not sure if adding support@forus.io in the footer is really wanted. Do they really need a form?

**Comment by jamalv** [comment]
I think this one doesn't fit in the current scope. Putting it on hold for now, needs more discussion.

**Comment by GerbenBosschieter** [comment]
Adding support@forus.io is not the option thats right :P

**Comment by maxvisser** [comment]
If there is a figma proposal for:
1. adding a field to CMS for e-mail contact details of implementation (could also be used for #535 and making footer generic)
2. A page with a contact form on webshop and contact details

I think this is a fix for aswell an issue for user testing potjeswijzer (easy way to contact owner of site)

This part of the footer should be configurable:
![image](https://user-images.githubusercontent.com/10818702/96606105-30360200-12f7-11eb-9790-36ef61a209fe.png)

I would only allow users to mail (no public form); example from stadjerspas website:
![image](https://user-images.githubusercontent.com/10818702/96606165-45129580-12f7-11eb-9b61-1140eb998a0a.png)

**Comment by maartenfv** [comment]
What is the status here?

**Comment by maxvisser** [comment]
No more then comments and discussion, waiting on a decision (proposal)

**Comment by jamalv** [comment]
mailto link was accepted by stakeholders, added this insight to productboard: https://app.productboard.com/inbox/notes/15569035
